packages/extension-api-explorer/src/ExtensionApiExplorer.tsx

```diff
@@ -128,7 +133,13 @@ export const ExtensionApiExplorer: FC = () => {
       <RunItProvider sdk={sdk} configurator={configurator} basePath="">
         <>
           {specs ? (
-            <ApiExplorer specs={specs} envAdaptor={extensionEnvAdaptor} />
+            <ApiExplorer
+              specs={specs}
+              envAdaptor={extensionEnvAdaptor}
+              setVersionsUrl={runItNoSet}
+              // TODO We need expand/collapse side nav for the headless extension before we enabled this
```

Comment:
is there a ticket for this?

Reply:
Not yet. Wanted to discuss with you and @bryans99

Reply:
and @jhardy

Reply:
I'm in favor of the expand/collapse sidebar hamburger alternative. Created a ticket yet?